### PR TITLE
Ethan

### PR DESCRIPTION
**Pull request by XLuna24**
Add comments to blog page

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -35,7 +41,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>xluna-step-2020</deploy.projectId>
```
Comment (kirantumkur) [inline portfolio/pom.xml:44]: Move this file change to a different pull request. this belongs to week 2 work.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -23,10 +26,29 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private List<String> fact;
+    
+    @Override
```
Comment (kirantumkur) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:31]: check if this can be a const.

A const is initialized at compile time. Init() are functions that should be used as a the last resort. This leads to dependencies in control flow, which is easier if avoided.

File: portfolio/pom.xml (hunk 1 above)
Comment (XLuna24) [reply portfolio/pom.xml:44]: This was changed in branch clyde. but since its not merge the change also appears here.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (XLuna24) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: changed public void init() => public DataServlet()
